src/style.css

```
@@ -1 +1,40 @@
-/* add styles here */
+body  {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--5 :nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(n) {
  background-image: url(images/star-active.svg);
}
```
background repeat already settled in .stars__star selector.
combine all selectors with same styles in one block with comma separator